docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building a go binary that includes golang-fips patches
+
+## Env variables
+
+`GOEXPERIMENT=strictfipsruntime` helps ensure a fips compliant binary is build.  Note that this is functionally equivalent of
+
+```
+GO_BUILDTAGS+="goexperiment.strictfipsruntime"
+```
+
+You can export more Go flags like `GO_GCFLAGS="-N -l"` to add symbols in the final binary.
```

Comment:
I would remove this, as it's documented in `./make.bash` and isn't unique to this fork.

Reply:
This also disabled some classes of optimizations so it isn't recommended for normal builds.

Reply:
Fixed